.gitignore

```diff
@@ -5,4 +5,5 @@
 # https://digitalfortress.tech/tricks/creating-a-global-gitignore/
 
 build
+.vscode
```

Comment:
Please add this to **your** global `.gitignore`. While VSCode is the defacto default editor for Go, we don't want to keep its config in the project `.gitignore` file (it is not files generated by Mainflux or for the Mainflux project needs).